snakemake/executors/ga4gh_tes.py

```diff
@@ -0,0 +1,381 @@
+__author__ = "Johannes Köster"
+__copyright__ = "Copyright 2015-2020, Johannes Köster"
+__email__ = "koester@jimmy.harvard.edu"
```

Comment:
Please add your names and emails here.

Reply:
Is this required? We wouldn't mind if it stays as it is.

Reply:
But that is your work, at least change into `__authors__` and add a list with the three of us instead of my name only.

Reply:
ok